zcash_client_sqlite/src/scan.rs

```diff
@@ -57,6 +57,38 @@ struct WitnessRow {
 pub fn scan_cached_blocks<P: AsRef<Path>, Q: AsRef<Path>>(
     db_cache: P,
     db_data: Q,
+) -> Result<(), Error> {
+    scan_cached_block_batch(db_cache, db_data, std::i32::MAX)
+}
+
+/// Scans new blocks added to the cache for any transactions received by the tracked
+/// accounts but does so in batches of limited size.
+///
+/// This function pays attention only to cached blocks with heights greater than the
+/// highest scanned block in `db_data`. Cached blocks with lower heights are not verified
+/// against previously-scanned blocks. In particular, this function **assumes** that the
+/// caller is handling rollbacks.
+///
+/// For brand-new light client databases, this function starts scanning from the Sapling
+/// activation height. This height can be fast-forwarded to a more recent block by calling
+/// [`init_blocks_table`] before this function.
+///
+/// Scanned blocks are required to be height-sequential. If a block is missing from the
+/// cache, an error will be returned with kind [`ErrorKind::InvalidHeight`].
+///
+/// # Examples
+///
+/// ```
+/// use zcash_client_sqlite::scan::scan_cached_blocks;
+///
+/// scan_cached_blocks("/path/to/cache.db", "/path/to/data.db");
```

Comment:
```suggestion
/// scan_cached_blocks_incremental("/path/to/cache.db", "/path/to/data.db");
```

Reply:
Stuck with one function approach and original name.

Addressed in ab100d3b35e2130e0b81631cd20585cde2ef9f75